great_expectations/expectations/metrics/map_metric_provider/map_condition_auxilliary_methods.py

```diff
@@ -109,26 +109,21 @@ def _pandas_map_condition_index(
             domain_records_df = domain_records_df[
                 domain_records_df[column_name].notnull()
             ]
-        domain_column_name_list.append(column_name)
 
-    # multi-column map expectations
-    elif "column_list" in accessor_domain_kwargs:
-        column_list: List[Union[str, sqlalchemy.quoted_name]] = accessor_domain_kwargs[
-            "column_list"
-        ]
-        verify_column_names_exist(
```

Comment:
This looks like a behavior change (deleting the verification that column names exist), should this maybe be in a separate PR?

Reply:
@anthonyburdi the new code verifies, making this here unnecessary (the tests confirmed this).

Reply:
@anthonyburdi Marking as resolved per our discussion.  Thanks.